### DISCUSSION TITLE
Cant figure out how to create custom schematic

### DISCUSSION THREAD

**Discussion by sdickerhof** [cyrilletuzi/vscode-angular-schematics#931] [Q&A] [answered]
When i click on the command Create a Custom Schematic nothing happens.  
I created a schematics folder in project and also installed @angular-devkit/schematics-cli just in case it uses it. 

What am i missing?  i have purchaesd the pro edition and want to create custom templates for components with my own list of default imports etc.

**Comment by cyrilletuzi** [comment]
Hello @sdickerhof,

Just sent you an email at the address with which you subscribed to the Pro edition.

There is indeed an issue with the "Create a custom schematic" button. Maybe because of a change in a recent VS Code update.

I will look into it as soon as possible.

**Comment by cyrilletuzi** [comment, marked as answer]
Hi @sdickerhof,

This is fixed in version 6.14. It may require 2 VS Code restarts (one for the extension to update, the second one to apply the update).
